spec.html

```diff
@@ -32409,6 +32409,24 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-makefullyear" type="abstract operation">
+        <h1>
+          MakeFullYear (
+            _year_: a Number,
+          ): an integral Number or *NaN*
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It returns the full year associated with the integer part of _year_, interpreting any value in the inclusive interval from 0 to 99 as a count of years since the start of 1900. For alignment with the proleptic Gregorian calendar, "full year" is defined as the signed count of complete years since the start of year 0 (1 B.C.).</dd>
+        </dl>
+        <emu-alg>
+          1. If _year_ is *NaN*, return *NaN*.
+          1. Let _truncated_ be ! ToIntegerOrInfinity(_year_).
+          1. If _truncated_ is in the inclusive interval from 0 to 99, return *1900*<sub>𝔽</sub> + 𝔽(_truncated_).
+          1. Return 𝔽(_truncated_).
```

Comment:
Returning the truncated value here is different from what the current steps do (i.e., they say `let _yr_ be _y_`, not `let _yr_ be _yi_`), so the commit message should maybe acknowledge that and say why it's okay.

Reply:
Fair enough; done.